variables.tf

```diff
@@ -1,5 +1,44 @@
-#variable "example" {
-#  type        = string
-#  default     = ""
-#  description = "This is just an example of a variable"
-#}
+variable "image_url" {
+  type        = string
+  default     = "public.ecr.aws/p5r4i7k7/sidekick:latest"
+  description = "Container Image"
+}
+variable "resource_name_prefix" {
+  type        = string
+  description = ""
+}
+variable "resource_name_suffix" {
+  type        = string
+  description = ""
+}
+
+variable "cloud_integration_name" {
```

Comment:
We call this `lacework_integration_name` in both the config and cloudtrail modules - do we want to change to that for consistency sake?